.gitignore

```diff
@@ -65,6 +65,7 @@ launch-*
 # emacs
 .auctex-auto
 *_flymake.hs
+.dir-locals.el
```

Comment:
Would we want to take this opportunity to add other ignores? Looking at https://github.com/github/gitignore/blob/main/Global/Emacs.gitignore, there could be a few, but I don't use emacs so you decide!

Reply:
Good remark, but any sufficiently configured emacs probably wouldn't create any of those in the project itself. Thanks in any case!